newdle/client/src/components/creation/timeslots/Timeline.js

```diff
@@ -211,13 +213,15 @@ function TimelineInput({minHour, maxHour}) {
           <div className={styles['candidates-group']} key={i}>
             {rowCandidates.map(time => {
               const slotProps = getCandidateSlotProps(time, duration, minHour, maxHour);
+              const participants = availability?.find(a => a.startDt === `${date}T${time}`);
               return (
                 <CandidateSlot
                   {...slotProps}
                   key={time}
                   isValidTime={time => !candidates.includes(time)}
                   onDelete={() => handleRemoveSlot(time)}
                   onChangeSlotTime={newStartTime => handleUpdateSlot(time, newStartTime)}
+                  text={(participants?.availableCount || 'No') + ' participants registered'}
```

Comment:
let's use proper i18n and pluralization here since it's available now :)

Reply:
Ah yes, we do have them now!